taxonomies/additives.txt

```diff
@@ -12989,7 +12989,7 @@ cs:E472e, Estery mono- a diglyceridů mastných kyselin s kyselinou mono- a diac
 da:E472e, Mono- og diacetylvinsyreestere af mono- og diglycerider af fedtsyrer
 de:E472e, Mono- und Diacetylweinsäureester von Mono- und Diglyceriden von Speisefettsäuren, Diacetylweinsäureglyceride, Diacetylweinsäureester von Mono- und Diglyceriden von Speisefettsäuren, E 472e
 el:E472e, Εστερες του μονο- και διακετυλοτρυγικου οξεος με μονο- και διγλυκεριδια λιπαρων οξεων
-es:E472e, Ésteres monoacetiltartárico y diacetiltartárico de monoglicéridos y diglicéridos de ácidos grasos, Éster monoacetiltartárico de mono- y diglicéridos de ácidos grasos
+es:E472e, Ésteres monoacetiltartárico y diacetiltartárico de monoglicéridos y diglicéridos de ácidos grasos, Éster monoacetiltartárico de mono- y diglicéridos de ácidos grasos,datem
```

Comment:
datem? what does it mean? the only product with it seems to be https://es.openfoodfacts.org/producto/7501000122332/pan-blanco-bimbo

Reply:
I wondered too, it's defined here: https://en.wikipedia.org/wiki/DATEM

Reply:
Sounds good. Can you put the DATEM entry on the list of English synonyms instead? English synonyms of additives are applied automatically to all languages.

Reply:
ok, done. There are entries in en: and fr: as well. Can there be removed?

Reply:
Sorry I wasn't clear, I meant putting DATEM in the en: line -> that makes it a synonym.

The lines starting with synonyms:en: are used to make synonym of words used in taxonomy entries.